mlir/utils/jenkins/Jenkinsfile.nightly-perf

```diff
@@ -20,19 +20,21 @@ pipeline {
         stage('Build MIOpenDriver') {
             steps {
                 dir('MIOpen') {
+                    sh 'mkdir install'
```

Comment:
Could you make this a more specific name such as "MIOpenLocalInstallDir" just so that it doesn't get confuse between whether it is a dir name or command.